builds/misc/images-release.yaml

```diff
@@ -0,0 +1,606 @@
+        displayName: "Functions Binding nuget package Sign"
+        inputs:
+          ConnectedServiceName: "Azure IoT Edge Code Sign 2"
+          FolderPath: $(Build.BinariesDirectory)/publish
+          Pattern: Microsoft.Azure.WebJobs.Extensions*.nupkg
+          inlineOperation: |
+              [
+                {
+                    "keyCode": "CP-401405",
+                    "operationSetCode": "NuGetSign",
+                    "parameters": [],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-401405",
+                    "operationSetCode": "NuGetVerify",
+                    "parameters": [],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-agent
+          name: "Edge Agent"
+          project: Microsoft.Azure.Devices.Edge.Agent.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-hub
+          name: "Edge Hub"
+          project: Microsoft.Azure.Devices.Edge.Hub.Service
+      - template: templates/image-linux.yaml
+        parameters: 
+          imageName: azureiotedge-simulated-temperature-sensor
+          name: "Temperature Sensor"
+          project: SimulatedTemperatureSensor
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics
+        displayName: 'Build Image - azureiotedge-diagnostics - amd64'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch armv7l &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch armv7l
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM32'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch aarch64 &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch aarch64
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM64'
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifacts to VSTS'
+        inputs:
+          PathtoPublish: '$(Build.BinariesDirectory)\publish'
```

Comment:
It can't find this path. Any ideas why?

https://msazure.visualstudio.com/One/_build/results?buildId=31581852&view=logs&j=7b5daa7d-adff-5805-f451-ad7193802469&t=17084497-9798-5a5f-c90c-e528464a9aae

Reply:
In the error message below, the last slash is backward instead of forward.

##[error]Publishing build artifacts failed with an error: Not found PathtoPublish: /home/vsts/work/1/b\publish

Reply:
Oh I forgot to convert to linux path thanks